src/models/data.py

```diff
@@ -596,8 +617,23 @@ def _calculate_latlons(self, x: xr.Dataset) -> Tuple[np.ndarray, np.ndarray]:
         return latlons, train_latlons
 
     def _calculate_static(self, num_instances: int) -> np.ndarray:
+        # make sure values are DROPPED!
+        assert self.static is not None
+        if self.ignore_vars is not None:
```

Comment:
Need to remove the pixel / global means for the ignore_vars from the static data

Reply:
Not done yet

Reply:
done in the lines below:
```
            include_vars = [
                v
                for v in self.static.data_vars
                if all([ign_v not in v for ign_v in self.ignore_vars])
            ]
            self.static = self.static[include_vars]
```